packages/core/src/sass/globals.scss

```diff
@@ -47,7 +47,7 @@ $uikit-maxwidth: 42em; //Answer to life, the universe, and everything (keeps thi
 $uikit-colour-Hero: #115361 !global; //color variables (yes! the actual spelling doesn't have a u)
 $uikit-colour-Primary: #880e48 !global;
 $uikit-colour-Background: mix(darken(#f0f3f5, 40), #f0f3f5, 50);
-$uikit-colour-Border: mix(black, #bebebe, 50) !global;
+$uikit-colour-Border: #bbbbbb !global;
```

Comment:
is it really necessary to use `!global` ?

Given this from docs:

> Variables are only available within the level of nested selectors where they’re defined. If they’re defined outside of any nested selectors, they’re available everywhere. They can also be defined with the !global flag, in which case they’re also available everywhere. 


Seems redundant. 

Also shouldn't we be using alternative `!default` so that globals can be overridden?